packages/mds-geography-service/repository/index.ts

```diff
@@ -58,12 +58,16 @@ class GeographyReadWriteRepository extends ReadWriteRepository {
 
   protected findGeographies = async (
     where: FindManyOptions<GeographyEntity>['where'],
-    { includeMetadata = false }: GetGeographiesOptions = {}
+    { includeMetadata = false, excludeGeographyJSON = false }: GetGeographiesOptions = {}
```

Comment:
It might be better to be more consistent in the options naming. Maybe:
```ts
{ includeMetadata = false, includeGeographyJSON = true }: GetGeographiesOptions = {}
```

Reply:
Happy to change that